packages/big-design/src/components/Search/spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { Search } from './Search';
+
+test('renders the search component', () => {
+  render(<Search value="Product" onChange={jest.fn()} onSubmit={jest.fn()} />);
+
+  const input = screen.getByLabelText('Search') as HTMLInputElement;
+
+  expect(screen.getByText('Search')).toBeInTheDocument();
```

Comment:
Is this the button you're trying to get? If so `screen.getByRole('button', { name: /search/i })` is a bit better

Reply:
Copied those over, but definitely a good catch! Pushed the changes.